_analyzers/index.md

```diff
@@ -15,16 +15,24 @@ redirect_from:
 
 # Text analysis
 
-When you are searching documents using a full-text search, you want to receive all relevant results and not only exact matches. If you're looking for "walk", you're interested in results that contain any form of the word, like "Walk", "walked", or "walking." To facilitate full-text search, OpenSearch uses text analysis.
+When you are searching documents using a full-text search, you want to receive all relevant results. If you're looking for "walk", you're interested in results that contain any form of the word, like "Walk", "walked", or "walking". To facilitate full-text search, OpenSearch uses text analysis.
 
-Text analysis consists of the following steps:
+The objective of the text analysis is to break down the unstructured free-text content of the source document into a sequence of terms, which are then stored in an inverted index. Subsequently when a similar text analysis is applied to a user's query, the resulting sequence of terms facilitates the matching of relevant source documents.
 
-1. _Tokenize_ text into terms: For example, after tokenization, the phrase `Actions speak louder than words` is split into tokens `Actions`, `speak`, `louder`, `than`, and `words`.
-1. _Normalize_ the terms by converting them into a standard format, for example, converting them to lowercase or performing stemming (reducing the word to its root): For example, after normalization, `Actions` becomes `action`, `louder` becomes `loud`, and `words` becomes `word`.
+From a technical point of view, the text analysis process consists of several steps, some of which are optional.
+
+1. Before the free-text content can be broken down into individual words it may be beneficial to treat it at the character level. The primary aim of this optional step would be to help the tokenizer (the subsequent stage in the analysis process) generating better tokens. This can include removal of markup tags (such as HTML) or handling a specific character patterns (like replacing emoji &#x1F642; with a text `:slightly_smiling_face:`).
```

Comment:
> ... it may be beneficial to "apply" it at the character level ...

I think we need to find a better formulation. What I try to say here is that it might be necessary or useful to "heal" the text at the character level. It is similar to "normalization" that I talk about later. We are not dealing with individual words yet but we may need to treat characters or patterns or characters.

My original formulation ("treat") was not good at all. But using only "apply" does not IMO do the right job as well. We need to say WHAT to apply. Or use a different word what has the meaning of "healing" or something like that ... sorry, I do not have better explanation, I know it is far from ideal.

Do you have any ideas please?

Reply:
How about "process" or "refine" the text at a character level? Does this work?

Reply:
Yep, I take:
> ... it may be beneficial to refine the text at a character level.